geth/peers/topicpool.go

```diff
@@ -52,7 +56,8 @@ type TopicPool struct {
 	poolWG sync.WaitGroup
 	quit   chan struct{}
 
-	running int32
+	running        int32
+	maxCachedPeers int
```

Comment:
i think `running` here is either 0 or 1. adding maxCachedPeers next to it might be confusing